Source/WebCore/loader/cache/AllowedFonts.h

```diff
@@ -41,7 +41,7 @@ enum class DownloadableBinaryFontAllowedTypes : uint8_t {
     None
 };
 
-bool isFontBinaryAllowed(std::span<const uint8_t>, DownloadableBinaryFontAllowedTypes);
-bool isFontBinaryAllowed(const void*, size_t, DownloadableBinaryFontAllowedTypes);
+bool isFontBinaryAllowed(std::span<const uint8_t>, DownloadableBinaryFontAllowedTypes, std::string url = "");
+bool isFontBinaryAllowed(const void*, size_t, DownloadableBinaryFontAllowedTypes, std::string url = "");
```

Comment:
If you really need a default value for url, then instead of `const String&`, it could be a `const String*` will `nullptr` as default. But make sure to check for null url.